### ISSUE TITLE
Add tests for trustee-operator

### ISSUE THREAD

**Issue by bpradipt** [confidential-containers/trustee-operator#34]
At least the following initial tests will be needed

- operator install and uninstall
- Trustee CRD creation, deletion
- secret release
- policy check (deny secret release due to policy deny)
- policy check (allow secret release due to policy allow)
- Update config, secret or policy via modifying the related K8s resources and verifying the update is reflected in the Trustee components
- scaling trustee pods and verifying secret release, policy checks, config updatesetc

**Comment by bpradipt** [comment]
Some options to explore

https://github.com/confidential-containers/operator/tree/main/tests/e2e
https://kuttl.dev/docs/#pre-requisites
https://kyverno.github.io/chainsaw/latest/

**Comment by sarroutbi** [comment]
Hello @bpradipt ... when referring to tests ... do you mean a GH action to perform the different proposals?

In https://github.com/latchset/tang-operator we had some tests where the operator was installed and uninstalled through operator-sdk ... is this the proposal?

**Comment by bpradipt** [comment]
@sarroutbi right. A basic test framework based on kuttl is added as part of GH action.

**Comment by bpradipt** [comment]
Closing this as we have basic tests and integration tests as well.
